src/jml/core.clj

```diff
@@ -586,22 +633,17 @@
 
 
 
-
-
-
 (run-multiple
- '(defn thing [Type/INT_TYPE Type/INT_TYPE Type/INT_TYPE]
+ '(defn thing2 [:int :int :int]
     (plus-int (arg 0) (arg 1)))
 
 
  ;; TODO: resolve types so this can be nicer
  ;; Also have function invokation in the language
- (list 'defn 'main [Type/VOID_TYPE]
-       (list 'print
-        (list 'invoke-static {:owner (Type/getType (Class/forName "thing"))
-                              :method (Method. "invoke" Type/INT_TYPE (into-array Type [Type/INT_TYPE Type/INT_TYPE]))}
-
-              1 2)))
```

Comment:
btw this example doesn't work, it always runs the old version of `thing` and fails first time around. It's because  `(Type/getType (Class/forName "thing"))` in line 601  is evaluated before class `thing` is compiled!